ports/libb2/vcpkg.json

```
@@ -1,7 +1,7 @@
 {
   "name": "libb2",
   "version-string": "0.98.1",
```

```suggestion
  "version": "0.98.1",
```
It should be version, please see https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md.